### ISSUE TITLE
Distributed events

### ISSUE THREAD

**Issue by DenuxPlays** [loco-rs/loco#1028]
## Feature Request

Currently we have Background Workers where one Job is only given to one Worker.
It would be great if we can fire events where one event is given to all workers.

I need to inform every server running when a specific entity is created.

**Describe the solution you'd like**

A system that works just like the workers but the event is published to every worker instead of only one.

**Describe alternatives you've considered**

There aren't really any alternatives in the rust ecosystem.

**Comment by jondot** [comment]
Sounds like some kind of pubsub requirement (subscriptions via Redis for example) doesn’t it?

**Comment by DenuxPlays** [comment]
> Sounds like some kind of pubsub requirement (subscriptions via Redis for example) doesn’t it?

Yes kinda but that both can be used simultaneously and controlled via code.
Just like Messages and Events in other frameworks like spring or symfony.
